applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -167,6 +167,15 @@ return view.extend({
 		o = s.taboption('advanced', form.Value, 'upnp_lease_file', _('UPnP lease file'))
 		o.placeholder = '/var/run/miniupnpd.leases'
 
+        	s.taboption('advanced', form.Flag, 'use_stun', _('Use STUN')).default = '0'
```

Comment:
Please change the following:
- Change `default` to `o.default = '0'` into a new line this is more common
- Add a dependency to `use_stun` for `stun_host` and `stun_port` so that this option are only shown if the flag `use_stun` is selected
- Update the commit message header to `luci-app-upnp: add STUN options`

Reply:
Fixing in an upcoming commit - thanks for the feedback